src/DacpacTool/PackageBuilder.cs

```diff
@@ -69,6 +69,13 @@ public void AddInputFile(FileInfo inputFile)
                 throw new ArgumentException($"Unable to find input file {inputFile}", nameof(inputFile));
             }
 
+            // Skip custom rules files
```

Comment:
It would be helpful to clarify _why_ we're doing this -- i.e. we're copying them to the output folder, and they'll be consumed from there. It's clear in the context of this PR, but in a couple of years we may look back on this and wonder why we'd want to skip certain files.

Reply:
I added more context to the comment